Makefile

```diff
@@ -107,12 +107,14 @@ generate-sanity-test:
 # Metalava & kotlin binary compatibility validator: check API
 .PHONY: check-api
 check-api:
+	./gradlew assembleRelease;
 	./gradlew checkApi;
 	./gradlew apiCheck;
```

Comment:
btw checkApi / apiCheck seems so weird =)

Reply:
I can drop some inline comments there, `checkApi` is the task name from metalava, `apiCheck` is the task name from kotlin binary compatibility validator 🙈

Reply:
Yeah I remember that - just it seems really confusing. Thought about renaming to the clear worded names like 
./gradlew checkApiMetalava 
./gradlew checkApiKotlinValidator

Reply:
we can rename metalava, but not Kotlin binary validator

Reply:
> we can rename metalava, but not Kotlin binary validator

But we should be able to make an alias instead of rename, right? E.g.

```
task chackApiKotlinValidator {
    dependsOn apiCheck
}
```

Reply:
@yunikkk  yes, but that would only add one extra task to the existing tasks, won't hide the `apiCheck` task. 

Actually if user execute `./gradlew tasks` they will get the detailed task description, imo it might worth the effort to create new wrapping tasks on existing task, a few lines of comment in the script should be sufficient to clarify things, wdyt?

Reply:
@yunikkk checking again, the task description seems not clear enough, so I added the suggested task names and added proper group names and descriptions, please take a look 👀